src/pharmpy/tools/external/results.py

```diff
@@ -1,10 +1,12 @@
 import importlib
 
 
-def parse_modelfit_results(model, path):
+def parse_modelfit_results(model, path, esttool=None):
```

Comment:
Is the `esttool` really necessary? When results are parsed using this function shouldn't the model always of the specific model type?